features/prompt-history/scoring-requests.mdx

```diff
@@ -8,14 +8,24 @@ Every PromptLayer request can be given an integer score 0-100.
 ![score](/images/score.png)
 
 To associate a score with a prompt, you can either do this visually from the dashboard or programmatically.
-
+Setting a name will allow you to set a named-score.
```

Comment:
By default, an individual score is named `default`. You can enrich a request with _multiple_ scores using "named scores" as shown below.